### PR TITLE
task#p001_login-markup

### PR DESCRIPTION
**Pull request by spideyinf**
localhost-3000-p001-login html 2017-10-23 09-21-26
p001-login-markup

### CODE REVIEW

File: source/p001-login.pug [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+html
+  +head(head.p001-login)
+
+  body
+    .p001-login
+      .container.h-100
+        .row.h-100
+          .col-md-5.h-100.d-flex.flex-column.justify-content-center.align-items-center
+            .mt-auto.mb-5.avatar-placeholder
+            div.p001-login__inputs
```
Comment (emily0604) [inline source/p001-login.pug]: remove div

File: source/sass/components/_ui-elements.scss [hunk 2]
```diff
@@ -16,3 +16,14 @@
 
   }
 }
+
+.avatar-placeholder {
+  background-image: url('#{$path-to-images}avatar-placeholder.png');
+    background-repeat: no-repeat;
```
Comment (emily0604) [inline source/sass/components/_ui-elements.scss]: please check indent

File: source/sass/pages/p001-login.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  background-image: url('#{$path-to-images}login-background.png');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  width: 100%;
+  height: 100vh;
+}
+
+
+.p001-login__inputs {
```
Comment (emily0604) [inline source/sass/pages/p001-login.scss]: using  nested loop

**Review by emily0604 — CHANGES_REQUESTED** [review]

File: source/p001-login.pug [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      .container.h-100
+        .row.h-100
+          .col-md-5.h-100.d-flex.flex-column.justify-content-center.align-items-center
+            .mt-auto.mb-5.avatar-placeholder
+            .p001-login__inputs
+              +input('', 'user')(name='username' placeholder='Username')
+              +input('', 'lock')(name='password' type='password' placeholder='Password')
+              .my-3.d-flex.justify-content-center
+                +checkbox('Remember me')(name='remember-me')
+              +button('login', 'anchor').btn-gradient-primary.btn-block
```
Comment (emily0604) [inline source/p001-login.pug]: not using gradient for name.

File: source/p001-login.pug [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+            a.mt-3(href='#') Forgot password?
+            p.mt-3 Not a member yet?
+              a(href='#')
+                u Sign up here
```
Comment (emily0604) [inline source/p001-login.pug]: u? what is u?

File: source/p001-login.pug (hunk 4 above)
Comment (emily0604) [reply source/p001-login.pug]: please check mixin button again . 
```
mixin button(label, link, icon)
  if typeof link !== 'undefined' && link === 'anchor'
    a(href='#').btn&attributes(attributes)
      if typeof icon !== 'undefined' && icon !== ''
        i(class=icon).btn__icon
      span.text= label
  else
    button.btn&attributes(attributes)
      if typeof icon !== 'undefined' && icon !== ''
        i(class=icon).btn__icon
      span.text= label
```

**Review by emily0604 — CHANGES_REQUESTED** [review]

File: source/p001-login.pug (hunk 5 above)
Comment (spideyinf) [reply source/p001-login.pug]: This is a html tag underline

**Review by emily0604 — APPROVED** [review]

**Pull request merged**